docs/source/content.rst

```diff
@@ -28,7 +25,7 @@ If we want to create a new document within an existing folder, we send a POST re
 
   .. code-block:: http-request
 
-    POST /folder HTTP/1.1
+    POST /{{folder_name}} HTTP/1.1
```

Comment:
@prakharjoshi I'm afraid this won't work since we moved to auto-generate the examples. We haven't been able to replace this everywhere yet though.